go/vt/sqlparser/goyacc/goyacc.go

```diff
@@ -166,11 +167,11 @@ var prefix string // name prefix for identifiers, default yy
 var allowFastAppend bool
 
 func init() {
-	flag.StringVar(&oflag, "o", "y.go", "parser output")
-	flag.StringVar(&prefix, "p", "yy", "name prefix to use in generated code")
-	flag.StringVar(&vflag, "v", "y.output", "create parsing tables")
-	flag.BoolVar(&lflag, "l", false, "disable line directives")
-	flag.BoolVar(&allowFastAppend, "fast-append", false, "enable fast-append optimization")
+	pflag.StringVar(&oflag, "o", "y.go", "parser output")
+	pflag.StringVar(&prefix, "p", "yy", "name prefix to use in generated code")
+	pflag.StringVar(&vflag, "v", "y.output", "create parsing tables")
+	pflag.BoolVar(&lflag, "l", false, "disable line directives")
```

Comment:
for all these, i would suggest something like:

```go
pflag.StringVarP(&oflag, "output", "o", "y.go", "parser output")
// similar long-opt names for the others (e.g. --name-prefix, --parsing-tables, --disable-line-directives)
```

this let's you do _either_ `goyacc --output "somepath.go"` or `goyacc -o somepath.go` (and for bools, you can jam them together, e.g. (assuming you have `-a` for `--allow-fast-append`) `goyacc -alo "somepath.go"`)

Reply:
Fixed!